discord_thread_summarizer_using_openai/Readme.md

```diff
@@ -0,0 +1,36 @@
+# Summarize Forum Channel threads in Discord using OpenAI
+
+## Use case
+Read and summarize Discord Forum Channel threads that has activity within the last 24 hours using OpenAI GPT 3.5-turbo model.
+
+## Prerequisites
+* OpenAI account
+* Discord Bot with Read access to a Discord server
+* Channel ID of the Discord Forum Channel
+
+### Setting up OpenAI account
+1. Create an [OpenAI](https://platform.openai.com/) account.
+2. Go to **View API keys** and create a new secret key.
+
+### Setting up the Discord Bot
+1. Follow the steps mentioned in Step 1 of the [Discord Developer Portal](https://discord.com/developers/docs/getting-started#step-1-creating-an-app) to create a new App and a Discord Bot with `Read Messages/View` Channels.
```

Comment:
Is this sentence complete? Is the last part correct/complete?

Reply:
Yes this is correct

Reply:
Aren't these privileges?

Reply:
Fixed